.xe/specs/blueprint/spec.md

```diff
@@ -12,153 +12,137 @@ dependencies: []
-  - Outcome: Clear phase assignments guide adoption timing
+- As a **Framework User**, I need reusable playbooks and templates to adopt Catalyst's workflow in my own projects
+  - Outcome: Install via npm, run initialization, get immediate access to context engineering and spec-driven workflows
 
 ## Success Criteria
 
-- All 27 features documented with unique IDs, names, descriptions, phase assignments, dependencies, complexity, and priority
-- Feature dependency graph is acyclic (no circular dependencies)
-- Features are organized into phase-relative tiers (1.1, 1.2, 2.1, etc.)
-- Phase 1 features have detailed specifications; Phase 2-5 features have high-level descriptions
-- Complexity distribution is balanced (not all Large features in one tier)
-- Every feature has clear 1-2 sentence scope description
-- Rollout plan tracks feature implementation status across all phases
+- Successfully package and distribute Catalyst as npm package for reuse across multiple projects
+- 90%+ adherence to standardized templates and playbooks across all implementations
+- 100% of deliverables pass human review at spec, plan, and task checkpoints before implementation
+- Reduce time from feature concept to production-ready code by 50% compared to manual workflows
+- Achieve 80% autonomous execution between human checkpoints while maintaining quality standards
+- Support projects serving millions of monthly active users with enterprise-scale reliability, security, and scalability
+- Enable both Claude Code and GitHub Copilot integration with extensibility for future AI platforms
 
 ## Design principles
 
-- **Phase-driven prioritization**
-  > Features are assigned to phases based on strategic priorities from product.md (POC → Mainstream → Innovation → Platform → Enterprise → Scale). Phase assignments drive implementation sequencing and help users understand feature maturity.
+- **Context over repetition**
+  > Store project context centrally in `.xe/` directory so AI reads context once instead of receiving repeated instructions. Context files (product.md, architecture.md, engineering.md) provide single source of truth for vision, patterns, and principles.
 
-- **Dependency-first ordering**
-  > Within each phase, features are ordered by dependencies first (can't build X without Y), then by business value and risk. This prevents implementation order errors and enables parallel work on features in the same tier.
+- **Human checkpoints at deliverables**
+  > AI executes autonomously between checkpoints but requires human approval at spec, plan, and tasks milestones. This prevents AI hallucinations while maintaining velocity by avoiding micro-management.
 
-- **Living specification**
-  > The blueprint is a living document that tracks feature implementation status and can be updated as product strategy evolves. Phase transition checkpoints allow re-blueprinting before starting new phases.
+- **Spec-driven development**
```

Comment:
This doesn't sound like a principle. It may be what we're doing, but I don't think it's a principle.